app/assets/stylesheets/color_themes/_color_theme_override.scss

```diff
@@ -1,18 +1,5 @@
 /* Raw CSS */
 body {
-  a,
-  a.tag,
-  .btn-link,
-  #main_stream .stream-element > .media a.author-name,
-  #hovercard h4 a,
-  .stream-element .from a.self {
-    color: $link-color;
-
-    &:hover, &:focus {
-      color: darken($link-color, 10%);
-    }
```

Comment:
Yes, the theme changed minimal, but it's now unified with other themes. I think that some of the effects that are now removed were done "by accident", because simply all links were overridden the same way.